src/styles/blocks/card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-bottom: 46px;
+  margin-right: 48px;
+  &_product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 4px;
+    margin-block-start: 0;
```

Comment:
Do we need ` margin-block-start` here?

Reply:
I have default value of 'margin-block-start: 1em' in h3 tag default styles, and it moving my image in wrong way, but I think I can remove 'margin-block-start: 0' to normalize.css for only h3 tag